src/api/app/assets/stylesheets/webui2/requests.scss

```diff
@@ -31,3 +31,19 @@
 .request-flag-declined {
   @extend .text-danger;
 }
+
+#request-history {
+  .media .media-body {
+
+    // hyphens isn't supported on Desktop chrome
+    // https://caniuse.com/#search=hyphens
+    @supports(hyphens:auto) {
```

Comment:
Can you please check all the occurrences of `hyphens: auto`?

Reply:
I also just noticed, we're missing a space here between the `:` and `auto`.

Reply:
> Can you please check all the occurrences of `hyphens: auto`?

is that related with the issue #8130 ?

Reply:
I will take care of this in another PR.